apps/web/utils/configurationHelper.ts

```diff
@@ -0,0 +1,109 @@
+import axios from 'axios';
+import https from 'https';
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
+
+const envFilePath = path.resolve(__dirname, "../.env");
+const envTmpFilePath = path.resolve(__dirname, '../.env.tmp');
+
+// read .env file & convert to array
```

Comment:
We usually don't write comments in the app, if the code is complex we move it into a function and explain what the code does by having a descriptive function name.